dask/base.py

```diff
@@ -648,7 +649,10 @@ def tokenize(*args, **kwargs):
     """
     if kwargs:
         args = args + (kwargs,)
-    return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
+    time_taken, result = get_wall_time(lambda: tuple(map(normalize_token, args)))
+    if time_taken > 2: # seconds
+        warnings.warn("tokenize is slow: set name=False to avoid hashing")
```

Comment:
Should this also mention that some data is hashed faster than other data? The test case `test_arr_hashing` (modified from @qwitwa's test case in https://github.com/dask/dask/issues/4275) shows that putting python objects in a numpy array forces `tokenize` to use `pickle`.

Reply:
Should probably be a `RuntimeWarning.`

I think we should also include

* The time it took to hash the data
* The key and value in the config to set the threshold